games-action/supertuxkart/supertuxkart-1.2.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="3D open-source go-kart racing game"
+DESCRIPTION="3D open-source go-kart racing game with an extensive variety of characters, tracks, and modes to play."
+HOMEPAGE="https://supertuxkart.net"
+COPYRIGHT="2020 SuperTuxKart Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/supertuxkart/stk-code/releases/download/$portVersion/supertuxkart-$portVersion-src.tar.xz"
+CHECKSUM_SHA256="052edf0afdbeb99583fe8676fb0ab80ecb6103fb88b7540f858d1b5fa1297d37"
+SOURCE_FILENAME="supertuxkart-$portVersion-src.tar.xz"
+SOURCE_DIR="SuperTuxKart-$portVersion-src"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+ supertuxkart$secondaryArchSuffix = $portVersion
+ app:SuperTuxKart$secondaryArchSuffix
+ "
+
+REQUIRES="
+ haiku$secondaryArchSuffix
+#lib:libangelscript$secondaryArchSuffix
+ lib:libcurl$secondaryArchSuffix
+ lib:libenet$secondaryArchSuffix
+ lib:libfreetype$secondaryArchSuffix
+ lib:libGL$secondaryArchSuffix
+ lib:libglu$secondaryArchSuffix
+ lib:libglew$secondaryArchSuffix
+ lib:libharfbuzz$secondaryArchSuffix
+ lib:libjpeg$secondaryArchSuffix
+ lib:libopenal$secondaryArchSuffix
+ lib:libogg$secondaryArchSuffix
+ lib:libpng$secondaryArchSuffix
+ lib:libsdl2_2.0$secondaryArchSuffix
+ lib:libsqlite3$secondaryArchSuffix
+#lib:libsquish$secondaryArchSuffix
+ lib:libssl$secondaryArchSuffix
+ lib:libvorbis$secondaryArchSuffix
+ lib:libz$secondaryArchSuffix
+ "
+
+BUILD_REQUIRES="
+ haiku${secondaryArchSuffix}_devel
+#devel:libangelscript$secondaryArchSuffix
+ devel:libcurl$secondaryArchSuffix
+ devel:libenet$secondaryArchSuffix
+ devel:libfreetype$secondaryArchSuffix
+ devel:libGL$secondaryArchSuffix
+ devel:libglu$secondaryArchSuffix
+ devel:libglew$secondaryArchSuffix >= 2
+ devel:libharfbuzz$secondaryArchSuffix
+ devel:libjpeg$secondaryArchSuffix
+ devel:libpng$secondaryArchSuffix
+ devel:libopenal$secondaryArchSuffix
+ devel:libogg$secondaryArchSuffix
+ devel:libsdl2$secondaryArchSuffix
+ devel:libsqlite3$secondaryArchSuffix
+#devel:libsquish$secondaryArchSuffix
+ devel:libssl$secondaryArchSuffix
+ devel:libvorbis$secondaryArchSuffix
+ devel:libz$secondaryArchSuffix
+ "
+
+BUILD_PREREQUIRES="
+ cmd:cmake
+ cmd:gcc$secondaryArchSuffix
+ cmd:ld$secondaryArchSuffix
+ cmd:make
+ cmd:pkg_config$secondaryArchSuffix
+ "
+
+BUILD()
+{
+ mkdir -p build
+ cd build
+
+ cmake .. \
+	-DCMAKE_BUILD_TYPE=STKRelease \
+	-DCUSTOM_BINDIR="$appsDir/SuperTuxKart" \
+	-DSTK_INSTALL_DATA_DIR=$dataDir/SuperTuxKart \
+	-DSTK_INSTALL_BINARY_DIR=$appsDir \
+	-DCMAKE_INSTALL_PREFIX=$prefix \
+	-DCMAKE_INSTALL_LIBDIR=$libDir \
+	-DBUILD_RECORDER=OFF \
+	-DBUILD_SHARED_LIBS=OFF \
+	-DUSE_CRYPTO_OPENSSL=OFF \
+	-DUSE_IPV6=OFF \
+	-DUSE_SYSTEM_ENET=ON \
+	-DUSE_SYSTEM_GLEW=OFF \
```

Comment:
This recipe bundles glew's libraries, so using SuperTuxKart's bundled libraries doesn't make much sense here.

Reply:
If system libs are available they should be used (if it doesn't brake the build), no need to turn the switch OFF (or mention them in the recipe) on some of them, some need to be disabled though